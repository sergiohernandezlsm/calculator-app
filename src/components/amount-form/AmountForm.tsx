import Form from 'react-bootstrap/Form';
import Row from 'react-bootstrap/Row';
import Col from 'react-bootstrap/Col';
import styles from './AmountForm.module.scss';

const AmountForm = () => {
  return (
    <div className={styles.formWrapper}>
      <Form>
        <Form.Group className="mb-3" controlId="exampleForm.ControlInput1">
          <Row>
            <Col>
              <Form.Label>Amount requested</Form.Label>
            </Col>
          </Row>
          <Row>
            <Col xs={8}>
              <Form.Control type="number" placeholder="Amount" />
            </Col>
            <Col xs={4}>
              <Form.Text className="text-muted">
                (in Â£)
              </Form.Text>
            </Col>
          </Row>
        </Form.Group>
        <Form.Group className="mb-3" controlId="exampleForm.ControlInput1">
          <Row>
            <Col>
              <Form.Label>Duration</Form.Label>
            </Col>
          </Row>
          <Row>
            <Col xs={8}>
              <Form.Control type="number" placeholder="Duration" />
            </Col>
            <Col xs={4}>
              <Form.Text className="text-muted">
                (in months)
              </Form.Text>
            </Col>
          </Row>
        </Form.Group>
      </Form>
    </div>
  )
}

export default AmountForm