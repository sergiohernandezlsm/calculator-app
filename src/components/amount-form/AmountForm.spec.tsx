import { shallow } from 'enzyme';
import AmountForm from './AmountForm';

describe('AmountForm Component', () => {
  it('renders <AmountForm /> component', () => {
    const wrapper = shallow(<AmountForm />);
    expect(wrapper).toMatchSnapshot();
  });
});