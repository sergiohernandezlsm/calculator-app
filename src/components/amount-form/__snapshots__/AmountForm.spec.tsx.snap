// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AmountForm Component renders <AmountForm /> component 1`] = `ShallowWrapper {}`;
